src/components/Demo/DemoCreate/DemoCreateIssue/index.tsx

```diff
@@ -75,7 +76,11 @@ export const DemoCreateIssue: FunctionComponent = () => {
                 .
               </p>
               <Link to="/demo/verify">
-                <Button className="bg-cerulean text-white rounded hover:bg-cerulean-500">Test Your File Now</Button>
+                <Banner
+                  className="text-left"
```

Comment:
think need to pass in `to`, cos currently will go to contact page, which is incorrect